tests/ArgentPonyWarcraftClient.Extensions.DependencyInjection.Tests/ArgentPonyWarcraftClient.Extensions.DependencyInjection.Tests.csproj

```diff
@@ -1,8 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>netcoreapp3.0</TargetFramework>
-
+        <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Thanks for taking care of that.  At some point, I may get this running on .NET 5.  That should be a small change (CircleCI configuration and target frameworks).

Reply:
Yeah something complained at me about "This isn't LTS" so I made that change super fast. But it would be great if we could upgrade to .NET 5 or even do a multi-target of `netcoreapp2.1;netcoreapp3.1;net5.0` to cover the major LTS versions at the very least.

Reply:
I created https://github.com/blizzard-net/warcraft/issues/169 to capture this idea.